modules/roles_profiles/manifests/profiles/mac_v3_signing.pp

```diff
@@ -127,6 +127,54 @@
                     ed_key_filename     => $user_data['ed_key_filename'],
                 }
             }
+
+            class { 'telegraf':
+                global_tags  => {
+                    workerType    => $worker_type,
+                    workerGroup   => $worker_group,
+                    provisionerId => 'scriptworker-prov-v1',
+                    role          => $role,
+                    workerId      => $facts['networking']['hostname'],
+                },
+                agent_params => {
+                    interval          => '300s',
+                    round_interval    => true,
+                    collection_jitter => '0s',
+                    precision         => 's',
+                },
+                inputs       => {
+                    # current default telegraf monitors: system, mem, swap, disk'/', puppetagent
+                    system      => {},
+                    temp        => {},
+                    cpu         => {
+                        interval         => '60s',
+                        percpu           => true,
+                        totalcpu         => true,
+                        ## If true, collect raw CPU time metrics.
+                        collect_cpu_time => false,
+                        ## If true, compute and report the sum of all non-idle CPU states.
+                        report_active    => false,
+                    },
+                    mem         => {},
+                    swap        => {},
+                    disk        => {
+                        mount_points => ['/'],
+                    },
+                    diskio      => {},
+                    procstat    => {
+                        interval => '60s',
+                        exe      => '/builds/scriptworker/bin/scriptworker',
+                    },
+                    procstat2   => {
+                        plugin_type => 'procstat',
+                        interval    => '60s',
+                        pattern     => 'tools/release/signing/signing-server.py',
+                    },
+                    puppetagent => {
+                        location => '/opt/puppetlabs/puppet/cache/state/last_run_summary.yaml',
```

Comment:
Haven't we disabled puppet agent runs in #164 ? I can't merge that other PR, which is why it's not active yet.

Reply:
I'll merge that PR #164 now, and then we can see the real diff for scriptworkers.

Reply:
I think this code block is all new, so the diff won't change. Puppet agent shouldn't be running, in any case, but I don't know how this particular check works - is it looking for the age of the last run?

Reply:
Here's the readme and code for the pupppetagent plugin: https://github.com/influxdata/telegraf/tree/master/plugins/inputs/puppetagent
It won't alert itself on anything, but reads in most of the values from the yaml file. From that, we can see the age of the last run and success/failure.